frontend/src/PagesAdmin/RecruitmentApplicantAdminPage/components/RecruitmentInterviewNotesForm/RecruitmentInterviewNotesForm.tsx

```diff
@@ -76,13 +107,22 @@ export function RecruitmentInterviewNotesForm({ initialData, interviewId }: Recr
               <FormItem>
                 <FormLabel>{t(KEY.recruitment_interview_notes)}</FormLabel>
                 <FormControl>
-                  <Textarea
-                    {...field}
-                    onBlur={(newNotes) => {
-                      field.onBlur(); // Call the default onBlur handler from react-hook-form
-                      handleNotesChange(newNotes.target.value); // Call your custom function on blur
-                    }}
-                  />
+                  <div>
+                    {markdownState ? (
+                      <Textarea
+                        {...field}
+                        ref={textareaRef}
+                        className={styles.textBox}
+                        onBlur={(newNotes) => {
+                          field.onBlur(); // Call the default onBlur handler from react-hook-form
+                          handleNotesChange(newNotes.target.value); // Call your custom function on blur
```

Comment:
Overflødige kommentarer som kan fjernes. Kommentarer burde forklare _hvorfor_, ikke _hva_ – med mindre det er noe veldig komplisert da :~)

```suggestion
                          field.onBlur();
                          handleNotesChange(newNotes.target.value);
```